programs/ziti-edge-tunnel/package/deb/postrm.in

```diff
@@ -13,7 +13,13 @@ if [ "$1" = "purge" ]; then
 fi
 # End copied section
 
-if [ -L @SYSTEMD_UNIT_DIR@/@SYSTEMD_UNIT_FILE_NAME@ ]; then
+# delete the symlink to the executable that was created by the postinst scriptlet
+if [ -L /usr/bin/@CPACK_PACKAGE_NAME@ ]; then
```

Comment:
why would we not do this if the file isn't a symlink?

Reply:
We could delete it even if it's not a symlink. I estimated that it's prudent to only delete the file if it has the expected type, based on what what created by the packager's install operation. For example, if it's not a symlink, then it was probably deliberately placed there by the administrator. That reasoning is moot, arguably, because the install operation will always clobber the file, irrespective of type.